OrangeHRMTestFramework/PageObjects/OrangeHRM/EmployeeManagementPage.cs

```diff
@@ -0,0 +1,17 @@
+﻿using OpenQA.Selenium;
+using OrangeHRMTestFramework.Common.WebElements;
+using OrangeHRMTestFramework.PageObjects.OrangeHRM.DataGrids;
+using OrangeHRMTestFramework.PageObjects.OrangeHRM.Filters;
+
+namespace OrangeHRMTestFramework.PageObjects.OrangeHRM
+{
+    public class EmployeeManagementPage : BaseOrangePage
+    {
+        public BasicDataGrid BasicDataGrid => new BasicDataGrid();
```

Comment:
Please place properties after fields. And if there are other properties, please place on first place properties which represent elements, then - "modules" (grids, filters, etc). Please check in other classes
JFYI - ordering in class
Constants
Fields
Properties
Constructors
Public Methods
Protected Methods
Private methods

Reply:
Good point, thank you, I hope I will finally remember this rule someday 🥲 Fixed.